client/src/components/v2/CameraRecording.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Button, Card, Col, Row,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { startRecording, stopRecording, getPrettyDeviceName } from 'api/v2/camera';
+import CameraRecordingStatus from './CameraRecordingStatus';
+import styles from './CameraRecording.module.css';
+
+/**
+ * @typedef {object} CameraRecordingProps
+ * @property {Array} devices Array of strings of device names
```

Comment:
This type can be explicitly declared as an array of strings. https://stackoverflow.com/questions/14611995/how-to-specify-an-array-of-objects-as-a-parameter-or-return-value-in-jsdoc

Reply:
Done
31facd8

I assume we're using JSDoc 3